main.go

```diff
@@ -185,6 +186,15 @@ func populateArgs(c *cli.Context) (
 	return
 }
 
+func stringify(input any) string {
```

Comment:
Can we move this method in util?
internal/util/util.go

Reply:
it was only used in one file hence kept it here for now, will it be okay if we move it to utility when more than one file uses it

Reply:
additionally, I won't be able to test flagStorage example as flagStorage is private to main package

Reply:
have moved Stringify to util package